src/test/ui/terminal-width/whitespace-trimming-2.stderr

```diff
@@ -0,0 +1,14 @@
+error[E0308]: mismatched types
+  --> $DIR/whitespace-trimming-2.rs:4:187
+   |
+LL | ...-> usize {
```

Comment:
this isn't too great. Not sure what to do about it though

Reply:
I think this is a pathological case and the current behavior is the best possible one given the constraints.